.github/workflows/ci.yml

```
@@ -98,7 +98,8 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
        matrix:
-           os: [windows-latest, ubuntu-20.04]
+           os: [ubuntu-20.04]
```

```suggestion
           os: [windows-latest, ubuntu-20.04]
```
@chrpetre , we recently implemented a fix for the pytest, which should resolve the issues encountered earlier. Could you please rerun it and let us know if it resolves the problem?

---

@Revathyvenugopal162, thanks the windows test can now be executed. However an other error comes after, about the project/package not found during the actual testing ModuleNotFoundError: No module named 'pytwin'
https://github.com/ansys/pytwin/actions/runs/11158065217/job/31014240875
any idea on that issue ?

the test works fine on Linux - https://github.com/ansys/pytwin/actions/runs/11158065217/job/31014405457

---

Yes, this issue arises because the project is using Poetry. We are currently working on a fix, and @RobPasMue is handling this. If you don’t mind, could we keep this project open for a little while longer so he can test the latest changes? We’ll make the necessary fixes and merge the PR as soon as possible. Thank you!